jupyterlab_wipp_plugin_creator/README.md

```diff
@@ -0,0 +1,116 @@
+# jupyterlab_plugin_creator
+
+![Github Actions Status](https://github.com/Kingstonshaw/jupyterlab_plugin_creator/workflows/Build/badge.svg)
```

Comment:
Should I do this? (https://github.com/LabShare/jupyterlab-extensions/workflows/Build/badge.svg)
Or this (https://github.com/LabShare/jupyterlab-extensions/workflows)
Or (https://github.com/LabShare/jupyterlab-extensions/)

I am generally confused by Github Actions, followed the guide [quickstart](https://docs.github.com/en/actions/quickstart) and tried various tips from stackoverflow but was never able to get it to run at all, i.e., shows the workflow results under 'Actions'. Even doing (https://github.com/Kingstonshaw/jupyterlab_wipp_plugin_creator/workflows) is 404. Any advice?

Reply:
To enable Github Actions, we would need to enable Actions on the main repo itself. For that we would need to have some workflow in .github/workflows in the *root* of the repo. Since we have a monorepo, it is more complicated to do, i.e. we would need to have a workflows for each extension with subpaths specified. I think it is probably better to remove all the actions code and links for now and add them back for all extensions in a separate PR after this one is merged